manim/mobject/coordinate_systems.py

```diff
@@ -607,7 +607,12 @@ def get_riemann_rectangles(
 
         # setting up x_range, overwrite user's third input
         if x_range is None:
-            x_range = self.x_range
+            if bounded_graph is None:
+                x_range = [graph.t_min, graph.t_max, None]
```

Comment:
`None` isn't necessary here since a third third value is added.
```suggestion
                x_range = [graph.t_min, graph.t_max]
```